source/fundamentals/indexes.txt

```diff
@@ -397,6 +397,34 @@ Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB s
 
 .. driver-content-end
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations on **clustered collections**.
+Clustered collections store documents ordered by the clustered index key
+value.
+
+To create a clustered index, specify the clustered index option with the
+``_id`` field as the key and the unique field as ``true`` when you
+create your collection.
+
+The following example creates a clustered index on the ``_id`` field in
+the ``vendors`` collection:
+
+.. code-block:: java
+   
+   MongoDatabase database = mongoClient.getDatabase("tea");
+   ClusteredIndexOptions clusteredIndexOptions = new ClusteredIndexOptions(Filters.eq("_id", 1), true);
```

Comment:
Since there is no API in the driver that is intended to construct a valid `Bson` key for `ClusteredIndexOptions`, we should not recommend users to use unrelated API solely because it so happens that the API creates a document valid for our situation (`Projections.include("_id")` also happens to produce a valid document). If we do this, then users may reasonably expect that specifying the output of other `Filters` methods here is allowed, but it's not. I think that we instead should write `new ClusteredIndexOptions(new Document("_id", 1), true)`.